packages/core/src/Stores/client-store.js

```diff
@@ -1029,21 +1029,18 @@ export default class ClientStore extends BaseStore {
         const is_samoa_account = this.root_store.ui.real_account_signup_target === 'samoa';
         let currency = '';
         form_values.residence = this.residence;
+
         if (is_maltainvest_account) {
             currency = form_values.currency;
             form_values.accept_risk = form_values.accept_risk || 0;
-            delete form_values.currency;
         }
+
         const response = is_maltainvest_account
             ? await WS.newAccountRealMaltaInvest(form_values)
             : await WS.newAccountReal(form_values);
+
         if (!response.error) {
             await this.accountRealReaction(response);
-            // Set currency after account is created
-            // Maltainvest only
```

Comment:
Question: was there any reason behind setting currency for MF accounts?

Reply:
As I got know after discussing with BE team the currency field had been just missed in BE while implementing. I haven’t been able to find any other reason.